frontend/src/store/modules/subscription.ts

```diff
@@ -55,10 +57,13 @@ export const useSubscriptionStore = defineStore("subscription", {
       );
       return daysBeforeExpire / total < 0.5;
     },
+    canTrial(state): boolean {
```

Comment:
From this check, can user start another free trial after the current one ends?

Reply:
No, he can only trial for once, unless he clears the settings table.
By default, the user should have a free plan subscription, so that he can start the free trial. After the trial ends, he always gets the expired subscription (same for manually uploading the license)